.github/workflows/create_rc_pr.yml

```diff
@@ -87,17 +96,18 @@ jobs:
                 WARNING: ${{ needs.find_release_branches.outputs.warning }}
               run: |
                 if [ -n "${{ needs.find_release_branches.outputs.warning }}" ]; then
-                  echo "CHANGES=$(inv -e release.check-for-changes -r "$MATRIX" "$WARNING")" >> $GITHUB_OUTPUT
+                  echo "CHANGES=$(inv -e release.check-for-changes -r "$MATRIX" "$WARNING" | tail -n 1)" >> $GITHUB_OUTPUT
                 else
-                  echo "CHANGES=$(inv -e release.check-for-changes -r "$MATRIX")" >> $GITHUB_OUTPUT
+                  echo "CHANGES=$(inv -e release.check-for-changes -r "$MATRIX" | tail -n 1)" >> $GITHUB_OUTPUT
                 fi
 
             - name: Create RC PR
-              if: ${{ steps.check_for_changes.outputs.CHANGES == 'true'}}
+              if: ${{ steps.check_for_changes.outputs.CHANGES == 'true' || env.IS_AGENT6_RELEASE == 'true' }}
```

Comment:
Wouldn't this condition mean that every time we run this action on the schedule dedicated for Agent 6, then we will try to create new RC, regardless if there are any new changes? Because `env.IS_AGENT6_RELEASE` is always true in this case.

Reply:
I see. Then it all makes sense 👍